### PR TITLE
Adding the API that will respond to Tags webhhok

### PR DESCRIPTION
**Pull request by iifawzi**
Hi, This PR adds the API that will respond to the tags webhook.
Before going to production we need to set a `secret-token` in the gitlab configs, and then update the `env`

This PR should be merged after #24 to avoid conflicts.

### CODE REVIEW

File: src/configs/.env [hunk 1]
```diff
@@ -4,4 +4,5 @@ API_PREFIX='/'
 LOG_FILE_PATH='./src/logs'
 MongoURI='mongodb://localhost:27017'
 GITLAB_READ_API='https://gitlab.com/api/v4/projects/4600360/repository/files'
-GITLAB_TAGS_API='https://gitlab.com/api/v4/projects/4600360/repository/tags'
\ No newline at end of file
+GITLAB_TAGS_API='https://gitlab.com/api/v4/projects/4600360/repository/tags'
+GITLAP_HOOK_TOKEN='OURGITLABTOKENHERE'
```
Comment (en3sis) [inline src/configs/.env]: Should we be tracking this file?

Comment (en3sis) [reply src/configs/.env]: Also there's a typo, it says GITLAP

File: src/components/icons/router.icons.ts [hunk 2]
```diff
@@ -1,7 +1,9 @@
 import express from 'express'
+import * as controller from './controller.icons'
 const router = express.Router()
 
 const prefix = '/icons'
+router.get('/newReleaseGitlab', controller.newRelease)
```
Comment (en3sis) [inline src/components/icons/router.icons.ts]: can we update this route as `.post` since its the endpoint that will be created after

**Comment by en3sis** [comment]
What endpoint do you want the Gitlab webhook to send the notification to, `/v2/icons/newReleaseGitlab` ?

File: src/configs/.env (hunk 1 above)
Comment (iifawzi) [reply src/configs/.env]: Shouldn't be tracked, resolved and the typo is fixed

EDIT: Github action is depending on these variables to start the server for testing `v1` apis, that's why it's tracked.

File: src/components/icons/router.icons.ts (hunk 2 above)
Comment (iifawzi) [reply src/components/icons/router.icons.ts]: Resolved.

**Comment by iifawzi** [comment]
> What endpoint do you want the Gitlab webhook to send the notification to, `/v2/icons/newReleaseGitlab` ?

Yes

**Comment by en3sis** [comment]
> > What endpoint do you want the Gitlab webhook to send the notification to, `/v2/icons/newReleaseGitlab` ?
> 
> Yes

I'll add it and send you the secret :P

**Pull request merged**